src/test/com/codingnomads/betty/logic/services/ProbabilityToWinServiceTests.java

```diff
@@ -0,0 +1,52 @@
+package com.codingnomads.betty.logic.services;
+
+import com.codingnomads.betty.logic.models.TeamProbabilityToWin;
+import com.codingnomads.betty.logic.models.TeamSentimentScore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ProbabilityToWinServiceTests {
+
+    @Test(expected = NullPointerException.class)
+    public void ifTeamSentimentScoreIsNull_throwANullPointerException() {
+        TeamSentimentScore teamSentimentScore = new TeamSentimentScore();
+        teamSentimentScore.setScore(null);
+
+        ProbabilityToWinService service = new ProbabilityToWinService();
+        service.getProbabilityToWinFromSentimentAnalysis(teamSentimentScore);
+
+    }
+
+    @Test
+    public void givenAPositiveTeamSentimentScore_aTeamProbabilityToWinBetween0And100IsReturned() {
+        TeamSentimentScore givenTeamSentiment = new TeamSentimentScore();
+        givenTeamSentiment.setScore(227);
+
+        ProbabilityToWinService service = new ProbabilityToWinService();
+
+        assertEquals(givenTeamSentiment.getScore() / 100,
+                (double) service.getProbabilityToWinFromSentimentAnalysis(givenTeamSentiment).getProbabilityToWin(),
+                0.0);
+    }
+
+    @Test
+    public void givenANegativeTeamSentimentScore_aTeamProbabilityOfZeroIsReturned() {
```

Comment:
same as above. sentiment score analysis is never negative

Reply:
Cool cool. If you agree with the above, I would change this test to make sure it expects an exception to be thrown. Let me know :)

Reply:
Changed to expect a custom exception being thrown

Reply:
![tenor-226680446](https://user-images.githubusercontent.com/32600398/56577903-cacc5c80-65fe-11e9-93b0-1c3fa09283f5.gif)